src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -104,7 +106,7 @@ export const MapRoom: React.FC<MapRoomProps> = ({
         <img className="maproom__image" src={room.image_url} alt={room.title} />
       )}
 
-      {!isCovertRoom && (
+      {!isCovertRoom && isLabeled && (
```

Comment:
It would be good to combine these into a single `const` above, then use that here to keep the logic cleaner within the JSX.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1632/commits/ca8fae8cf8aa67b941e899e162433062a81f9564